<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFormWidget</class>
 <widget class="QWidget" name="UserFormWidget">
  <property name="windowTitle">
   <string>User Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lastNameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="usernameLabel_2">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="confirmPasswordLabel">
       <property name="text">
        <string>Confirm Password</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="confirmPasswordLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="departmentLabel">
       <property name="text">
        <string>Department</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="departmentComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateProfilePushButton">
       <property name="text">
        <string>Update Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
